src/projectConfiguration/yaml/models.ts

```diff
@@ -35,3 +35,13 @@ export enum FunctionType {
     // azure = "azure",
     // gcp = "gcp",
 }
+
+export const entryFileFunctionMap = {
+    js: "index.mjs",
+    ts: "index.mjs",
+    swift: "index.swift",
+    python: "index.py",
+    dart: "index.dart",
+    kt: "index.kt",
+    go: "index.go",
+};
```

Comment:
```suggestion
export const entryFileFunctionMap = {
    js: "index.mjs",
    ts: "index.mjs",
    python: "index.py",
    go: "main.go",
};
```

Reply:
It's a bit confusing to have entry files for languages we are not supporting in production now.